rest-resources/src/main/java/de/unistuttgart/iaas/amyassist/amy/restresources/home/HomeResource.java

```diff
@@ -41,22 +43,27 @@
 @Path(HomeResource.PATH)
 public class HomeResource {
 
+	/**
+	 * the Path of this resource
+	 */
 	public static final String PATH = "home";
 	@Reference
 	private PluginManager manager;
 	@Reference
 	private SpeechInputHandler speechInputHandler;
+	@Reference
```

Comment:
`UriInfo` is not from our di, it is form the Rest Service, `@Context` is the correct Annotation here `javax.ws.rs.core.Context`.